src/test/java/com/github/fge/jsonpatch/PathParserTest.java

```diff
@@ -0,0 +1,149 @@
+package com.github.fge.jsonpatch;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertThrows;
+
+public class PathParserTest {
+
+    @Test
+    void should_parse_json_pointer_to_json_path() throws JsonPatchException {
+        // given
+        String path = "/user/1/name";
```

Comment:
Missing also tests for object within object, like:
`/store/expensive`

Reply:
Added test for embedded object using json pointer